lib/clusterManager.js

```diff
@@ -680,8 +680,14 @@ if (cluster.isMaster) {
         return resultHandler[0];
       },
       function(e) {
-        clusterLogger.warn("ZWED0167W - Error adding to the storage: " + e);
-      }
+        if (e.startsWith('Timeout call')) {
```

Comment:
We have three places we reject with 'Timeout call' yet we hardcode "Timeout call " on line 533. If that ever changes, those places would need changes too, this seems like a good spot to put a CONST